virttest/utils_v2v.py

```diff
@@ -1047,7 +1047,7 @@ def _check_log(pattern_list, expect=True):
             expected = 'expected' if expect else 'not expected'
             logging.info('Searching for %s log: %s' % (expected, pattern))
             compiled_pattern = re.compile(line)
-            search = re.search(compiled_pattern, log)
+            search = re.search(compiled_pattern, log, re.S)
```

Comment:
change to re.s -->flags=re.s ?, it looks like more meaningful.

Reply:
Done